requirements_dev.txt

```diff
@@ -9,4 +9,5 @@ mypy==0.931
 types-dataclasses==0.6.4
 types-requests==2.27.3
 typing-extensions==4.0.1
-flake8==4.0.1
\ No newline at end of file
+flake8==4.0.1
+responses==0.17.0
```

Comment:
Add a library to intercept requests so we can return data without hitting the network